index.html

```diff
@@ -15,6 +15,7 @@
 <body>
     <img src="header-background-mobile.png" alt="">
     
+    
```

Comment:
This whole part of the page should be wrapped in the `section` tag. Kindly do so. You can refer to [this](w3schools.com/html/html5_semantic_elements.asp) link to understand semantic tags better.
![image](https://user-images.githubusercontent.com/33268927/185549756-47e3da58-a232-41f3-ac94-9bb6105060c7.png)

Reply:
Implemented 👍